ui/components/app/signature-request-siwe/signature-request-siwe-header/index.scss

```diff
@@ -0,0 +1,26 @@
+.signature-request-siwe-header {
+  display: flex;
+  padding: 16px;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+
+  &__tooltip__container {
+    display: flex !important;
+  }
+
+  .account-list-item {
+    margin: 8px 0 0;
+
+    &__top-row {
+      align-items: center;
+      margin: 0;
+    }
+
+    &__account-name {
+      @include H5;
+
+      font-weight: 500;
+    }
```

Comment:
[REQ] For any style where you are reaching to add class names only to style the text to one of our Typography levels (H5, here) then you can use the <Typography> component instead of writing additional styles.

Reply:
This reuses existing components, so I'll leave the changes in as css